src/AyonUsdResolver/cache/resolverContextCache.cpp

```diff
@@ -90,6 +162,9 @@ resolverContextCache::getAsset(const std::string &assetIdentifier,
     if (assetIdentifier.empty()) {
         return asset;
     }
+    if (this->m_static_cache) {
+        return this->m_pinningFileHanlder->getAssetData(assetIdentifier);
```

Comment:
```suggestion
        return this->m_pinningFileHandler->getAssetData(assetIdentifier);
```
Haha, you can likely best refactor this in your IDE all at once instead of me trying to find them all here... :)

Reply:
Fixed via a commit.